Cargo.toml

```diff
@@ -1,17 +1,10 @@
 [workspace]
 resolver = "2"
 
-members = [
-    "api",
-    "common",
-    "agent",
-    "server",
-    "grpc",
-    "ank"
-    ]
+members = ["api", "common", "agent", "server", "grpc", "ank"]
```

Comment:
Not that it really matters that much, but do we need this change?

Reply:
```suggestion
members = [
    "api", 
    "common", 
    "agent", 
    "server", 
    "grpc", 
    "ank"
    ]
```

Reply:
We do not need this change. This is done by the autoformatting tool.